### PR TITLE
Update post update checklist

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update post update checklist
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | -
| How to test?      | -

image

### CODE REVIEW

File: views/templates/block/postUpdateChecklistBlock.html.twig [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        </div>
+    </div>
+
+    <div class="panel">
+        <div class="panel-heading">
+            {{ 'Next steps'|trans({}) }}
+        </div>
+        <ul>
+            <li>{{ 'Re-enable and check your modules one by one.'|trans({}) }}</li>
+            <li>{{ 'Make sure your store’s front office is working properly: try to create an account, place an order,  add a product, etc.'|trans({}) }}</li>
```
Comment (Quetzacoalt91) [inline views/templates/block/postUpdateChecklistBlock.html.twig:27]:
```suggestion
            <li>{{ 'Make sure your store\'s front office is working properly: try to create an account, place an order, add a product, etc.'|trans({}) }}</li>
```

File: views/templates/block/postUpdateChecklistBlock.html.twig [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <li>{{ 'Disable the maintenance mode in General settings > Maintenance.'|trans({}) }}</li>
+        </ul>
+    </div>
+
+    <div class="panel">
+        <div class="panel-heading">
+            {{ 'Troubleshooting'|trans({}) }}
+        </div>
+        <ul>
+            <li>{{ 'If some images don’t appear in the front office, try regenerating thumbnails in Preferences > Images.'|trans({}) }}</li>
```
Comment (Quetzacoalt91) [inline views/templates/block/postUpdateChecklistBlock.html.twig:37]:
```suggestion
            <li>{{ 'If some images don\'t appear in the front office, try regenerating thumbnails in Preferences > Images.'|trans({}) }}</li>
```

File: views/templates/block/postUpdateChecklistBlock.html.twig [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            {% if PS_AUTOUP_BACKUP %}
+                <li>{{ 'If something’s wrong, you can restore a backup with this module. Your backup is available at {admin}/autoupgrade/backup.'|trans({}) }}</li>
```
Comment (Quetzacoalt91) [inline views/templates/block/postUpdateChecklistBlock.html.twig:39]:
```suggestion
                <li>{{ 'If something\'s wrong, you can restore a backup with this module. Your backup is available at {admin}/autoupgrade/backup.'|trans({}) }}</li>
```

File: views/templates/block/postUpdateChecklistBlock.html.twig [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            {% endif %}
+            <li>{{ 'If you can’t access your back office, try enabling the debug mode manually in config/defines.inc.php by setting _PS_MODE_DEV_ to true.'|trans({}) }}</li>
```
Comment (Quetzacoalt91) [inline views/templates/block/postUpdateChecklistBlock.html.twig:41]:
```suggestion
            <li>{{ 'If you can\'t access your back office, try enabling the debug mode manually in config/defines.inc.php by setting _PS_MODE_DEV_ to true.'|trans({}) }}</li>
```

**Review by Quetzacoalt91 — DISMISSED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by ga-devfront — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @M0rgan01 ,

Works as expected. tested the following :

1.7.8.11 to 8.1.7 ✅ 
Screenshot 2024-08-01 at 14 26 20

8.0.5 to 8.1.7 to 9.0.0 (rollback to each previous versions) ✅ 
Screenshot 2024-08-01 at 17 54 50

8.1.5 to 8.1.7 to 9.0.0 (rollback to each previous versions) ✅ 
Screenshot 2024-08-01 at 18 17 21

The new blocks and link are well displayed, the link opens in another page as expected :) 

Thanks! It is QA ✅

**Pull request merged**